### PR TITLE
Add lifebouy ads

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

**Comment by codiebeulaine** [comment]
Screenshot 2021-04-06 at 14 37 13

**Comment by codiebeulaine** [comment]
Screenshot 2021-04-06 at 14 36 54

**Review by rudigiesler — COMMENTED** [review]
180kb is quite large, the entire site is currently 126kb. Could we rather use an optimised jpeg for that image? 

Then for the <1025px target, the two symbols at the bottom look a bit strange, especially at larger widths. Maybe some spacing between the image and the following text heading, as well as ensuring that both images are the same height, and centred on the page, would help. 
image

image

**Comment by CourtneyMoxley** [comment]
One more comment here re the size of logos relative to the main advert image - please use this image as a guide on ratios. Please have the main ad as 2/3 and the logos as 1/3 
lifebuoy

File: higher_health/templates/login.html [hunk 1]
```diff
@@ -20,6 +21,12 @@ <h4>Thank you for coming forward and for contributing to the
               health of all citizens AND stopping the spread of the virus.🦠</h4>
           </div>
 
+          <div class="columns-child columns-child--excerpts">
+              <a class="lifebouy-ad" href="https://www.lifebuoy.co.za">
+                <img class="lifebouy-ad"  src="{% static 'images/POST_small.jpg' %}" alt="Higherhealth Logo" role="img">
```
Comment (rudigiesler) [inline higher_health/templates/login.html]: I see that there's no `images/POST_small.jpg` in this PR, only, `POST_1.jpg`. 

And then I think we should change the alt text to something more descriptive, like `Lifebouy Advertisement`

Comment (codiebeulaine) [reply higher_health/templates/login.html]: hmm, my changes don't seem to be coming through

Comment (codiebeulaine) [reply higher_health/templates/login.html]: @rudigiesler updated

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**